01_05_Conditions.txt

```diff
@@ -112,62 +111,61 @@ biofoam.
 fire or a continuous stream of bullets must overcome the psychological instinct
 to stay put and not get shot.
 
-    A target that is pinned is hunkered and cannot move for 2 full Rounds or 
```

Comment:
Should we specify that this hunker is an Immediate Reaction that costs Actions on their turn?